.gitlab/source_test/linux.yml

```diff
@@ -73,6 +73,19 @@ tests_deb-x64-py3:
     PYTHON_RUNTIMES: '3'
     CONDA_ENV: ddpy3
 
+tests_deb-x64-py-fast:
```

Comment:
Actually, on second thought should we have this allowed to fail?

Reply:
I can allow them to fail yes.